src/api/utils.ts

```diff
@@ -0,0 +1,48 @@
+import { Market } from "./types"
+
+export const mapMarketsData = (markets?: Market[]) => {
+  if (!markets) return undefined
+
+  return markets.map((i) => {
+    return {
+      ...i,
+      supplyApy: Number(i.supplyApy),
+      supplyVenusApy: Number(i.supplyVenusApy),
+      totalSupplyUsd: Number(i.totalSupplyUsd),
+      totalBorrowsUsd: Number(i.totalBorrowsUsd),
+      liquidity: Number(i.liquidity),
+    }
+  })
+}
+
+function compareMarkets(a: Market, b: Market) {
+  return b.supplyApy + b.supplyVenusApy - (a.supplyApy + a.supplyVenusApy)
+}
+
+export const getMarketsToRender = (markets?: Market[]) => {
+  if (!markets) return null
```

Comment:
This should probably return an empty array instead, so that when you map through whatever this function returns you don't have to check that it isn't `null`.

Reply:
changed to undefined